composer.lock

```diff
@@ -0,0 +1,140 @@
+{
```

Comment:
Naive question: Does this _have_ to be in here? I think I understand what you're trying to do with it, but having a `composer.lock` file in a front end, JavaScript-centric repository makes me squirm a bit.

Reply:
Now you know how we feel... ;)

https://github.com/department-of-veterans-affairs/va.gov-cms/blob/develop/package.json

Not sure yet. The .lock file could probably be ignored.

I'll keep trying to set up a pseudo-repo on our end so we don't need a composer.json on your end either.